localization/english/wc_physical_lifestyle_l_english.yml

```diff
@@ -0,0 +1,21 @@
+﻿l_english:
+ physical_lifestyle_name:0 "Physical"
+ physical_lifestyle_desc:0 "Focus on training your body and mind, on acquiring the strength to defeat your enemies."
+
+ stamina_name:0 "Resilient"
+ agility_name:0 "Trickster"
+ strength_name:0 "Might"
+
+ #Focus
+ physical_focus_1:0 "Toughness Focus"
+ physical_physical_focus_1_focus_desc:0 "Maintaining your body is vital to all pursuits."
+ physical_focus_1_effect_desc:0 "Focusing on Toughness grants:"
+ physical_focus_1_modifier:0 "[GetFocus('physical_focus_1').GetName]"
+ physical_focus_2:0 "Dexterity Focus"
+ physical_physical_focus_2_focus_desc:0 "Your enemies will not notice you until they're already dead."
```

Comment:
"... until it is already too late" Since you have the stasis trap as well, which doesn't kill.